cell_classification/segmentation_data_prep.py

```diff
@@ -465,7 +472,9 @@ def calculate_normalization_matrix(self, data_folders, selected_markers):
                 img = self.get_image(data_folder, marker)
                 if marker not in quantiles:
                     quantiles[marker] = []
-                quantiles[marker].append(np.quantile(img, self.normalization_quantile))
+                foreground = img[img > 0]
+                if np.sum(foreground) > 0:
+                    quantiles[marker].append(np.quantile(foreground, self.normalization_quantile))
```

Comment:
Did you check to see if this resolves the issue with the poor contrast? If not, we should open an issue about it and can later track it down.

Reply:
CD20:
Old:
![download](https://user-images.githubusercontent.com/22618181/200433867-ed799928-12ff-4eda-a933-929ffa961c5c.png)
New
![download](https://user-images.githubusercontent.com/22618181/200433879-776afc72-1f1a-42c5-8849-9d9063c481ab.png)

SMA:
Old:
![download](https://user-images.githubusercontent.com/22618181/200434327-b1d2cac2-6e6b-460e-b19a-93947d72cea9.png)

New:
![download](https://user-images.githubusercontent.com/22618181/200434395-dbd692ba-1b46-4d46-9255-993726ef66c4.png)

Reply:
Looks better I'd say. Most of the salt&pepper-like noise is reduced a lot.

Reply:
Agreed!